src/components/LayoutBase/LayoutBase.tsx

```diff
@@ -13,12 +18,26 @@ type LayoutBaseProps = {
 };
 
 export const LayoutBase = ({ children }: LayoutBaseProps) => {
+  const history = useHistory();
+
+  useEffect(() => {
+    history.location.pathname === '/' && handleNavbarBackground();
+  }, [history]);
```

Comment:
`history` is a pretty large object, we can put a primitive as a dependency
```suggestion
  }, [history.location.pathname]);
```